website/conferences/views.py

```diff
@@ -123,6 +127,18 @@ def add_poster_by_email(conference, message):
         presentation_type=message.conference_category.lower(),
         is_spam=message.is_spam,
     )
+    if node_created and user_created:
+        root_id = (node.get_addon('osfstorage')).root_node._id
+        mails.queue_mail(
+            to_addr=user.username,
+            mail=mails.WELCOME_OSF4M,
+            send_at=datetime.utcnow() + settings.WELCOME_OSF4M_WAIT_TIME,
+            user=user,
+            conference=conference.name,
+            fullname=user.fullname,
+            presentation='presentation',
+            fid=root_id
```

Comment:
I don't see `user` or `fid` being used in the `mails.WELCOME_OSF4M` template -- can they be removed?